src/App.tsx

```diff
@@ -17,18 +78,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                selectStatus={filteredStatus}
+                onSelectStatus={handleStatus}
+                query={query}
+                onQueryChange={handleQueryChange}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!todos.length ? (
+                <Loader />
+              ) : (
```

Comment:
Each condition should start from the new line
```suggestion
              {!todos.length 
                ? (
                  <Loader />
                ) 
                : (
```